test/hamiltonian_consistency.jl

```diff
@@ -51,6 +51,9 @@ function test_consistency_term(term; rtol=1e-4, atol=1e-8, ε=1e-6, kgrid=[1, 2,
         diff_predicted = 0.0
         for ik in 1:length(basis.kpoints)
             Hψ = ham.blocks[ik]*ψ[ik]
+            if ham.blocks[ik].operators isa DFTK.RealSpaceMultiplication
```

Comment:
This is a bit more tricky as
- This will never be true as operators is a Vector of Operators.
- Some terms add two operators.
Just check there is only a RealSpaceMultiplication and then run the check.